src/tests/ftest/server/metadata.py

```diff
@@ -296,25 +347,28 @@ def test_metadata_addremove(self):
         :avocado: tags=metadata_compact,metadata_addremove,test_metadata_addremove
         """
         self.create_pool()
+        if not self.run_dummy_metadata_workload():
+            self.fail("failed to run dummy metadata workload")
+
         self.container = []
         mean_cont_cnt = 0
         percent_cont = self.params.get("mean_percent", "/run/metadata/*")
 
         test_failed = False
         containers_created = []
-        for loop in range(10):
-            self.log.info("Container Create Iteration %d / 9", loop)
+        for loop in range(4):
```

Comment:
Please update test loop var to metadata.yaml.

Reply:
added this in the latest code on the branch.